### PR TITLE
Feat 589 search analytics

### PR DESCRIPTION
**Pull request by rbayet**
Implementation of #589

### CODE REVIEW

**Comment by romainruaud** [comment]
As seen together : 

* [ ] Fix the size of buckets

Right panel, two graphes : 
* [ ] pie chart to display "sessions with search", "session without search" in %age
* [ ] bar chart to display "session with search conversion rate", "session without search conversion rate"

**Comment by rbayet** [comment]
As seen together the graph API used on the native dashboard is soon to be deprecated
- > Warning: This API is deprecated and is scheduled to be turned off on March 14, 2019. Please use the actively maintained Google Charts API instead. 
(Source: https://developers.google.com/chart/image/)
- an issue has *just* [been created on M2 github](https://github.com/magento/magento2/issues/21599)

**Comment by rbayet** [comment]
Issue on Travis (a magento module could not get downloaded) has been fixed by clearing the Travis cache for the branch/PR.

File: src/module-elasticsuite-analytics/.gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+image: chialab/php:7.1
```
Comment (romainruaud) [inline src/module-elasticsuite-analytics/.gitlab-ci.yml]: This .gitlab-ci.yml file can probably be removed ?

Comment (rbayet) [reply src/module-elasticsuite-analytics/.gitlab-ci.yml]: Done

**Pull request merged**